University of Leicester/Leicester210.xml

```diff
@@ -0,0 +1,326 @@
+   <handDesc>
+<handNote xml:id="h1" script="Ethiopic">
+<seg type="script">
+<seg type="ink">Black, red</seg>
+<date notBefore="1350" notAfter="1450">Mid-14th mid-15th century (?)</date>
+<desc>Quite uniform handwriting, characterized by rounded letters, with broad and thick shapes. From <locus target="#87ra"></locus> to <locus target="#89vb"></locus> (text 4) the hand looks slightly different, less careful and in a smaller size. Peculiar and ancient features: the upper horizontal strokes of መ, ሠ, በ are slanted to the left; መ has sometimes angular loops; ዐ and ፬ have a triangular shape, downwards oriented; the loop of ሎ is attached to the body of the letter; ጵ has the 6th order marker set up on the vertical stroke at the top, oriented to the left (e.g. <locus target="#59rb"></locus>, <locus target="#69va"></locus>); ፪ has sometimes the loops attached; a few times it is with the bottom loop open (e.g. <locus target="#38ra"></locus>, <locus target="#38va"></locus>); መ and ሙ have sometimes the right loop detached from the ruled line (e.g. <locus target="#61va"></locus> l. 2, l. 3); ፮ is with the loop open, looking like 'compressed' ፯ (e.g. <locus target="#65ra"></locus>); marginal cues (numerals) for rubrication (e.g. <locus target="#35r"></locus>, <locus target="#81v"></locus>).
+</desc>
+<seg type="rubrication">Nomina sacra; a few lines, alternating with black lines, on the incipit page of text 1 (<locus target="#2r"></locus>), of text 2 (<locus target="#27r"></locus>), on the incipit of text 3 (<locus target="#53rb"></locus>) and text 4 (<locus target="#87ra"></locus>); the initial paragraph word (or words) on <locus target="#42rb"></locus>–<locus target="#42vb"></locus>, within text 2, and on <locus target="#88ra"></locus>–<locus target="#89rb"></locus>, within text 4; Trinitarian formula; name of the scribe and commissioner in the supplication formulas and subscriptions of text 3 (<locus target="#86vab"></locus>); Ethiopic numerals or their elements; elements of the punctuation signs. </seg>
+</seg>
+</handNote>
+   </handDesc>
+<decoDesc>
+<decoNote type="ornamentation" xml:id="d1">
+<locus target="#2r"></locus>
+<desc>Ornamental band (incipit page of text 1): coloured (red, yellow, green, and black), with one central and two lateral pendants reaching the bottom of the page; geometric and interlaces motifs.</desc>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d2">
+<locus target="#27r"></locus>
+<desc>Ornamental band (incipit page of text 2): coloured (red, yellow, green, and black), geometric and interlaces motifs.</desc>
+</decoNote>
+<decoNote type="ornamentation" xml:id="d3">
+<locus target="#53rb"></locus>
+<desc>Ornamental band (incipit of text 3): coloured (red, yellow, green, and black), geometric and interlaces motifs.</desc>
+</decoNote>
+</decoDesc>
+<additions>
+<list>
+<item xml:id="a1">
+<desc type="Record">Amharic note describing the content of the manuscript.</desc>
+<note>Black ink; recent hand</note>
```

Comment:
It is better to have the entire description inside `<desc>` and not to use `<note>`